app/models/service_template_provision_task.rb

```diff
@@ -111,6 +111,9 @@ def queue_post_provision
 
   def do_post_provision
     update_request_status
+
+    _log.info("Queuing post provision of #{self.class.name} id: #{id}, zone: #{zone}")
```

Comment:
Seems this message really belongs to `queue_post_provision`. Where do you get `#{zone}`? Should it be `my_zone`?